align/schema/constraint.py

```diff
@@ -1326,6 +1326,19 @@ class DoNotRoute(SoftConstraint):
     _upper_case = types.validator('nets', allow_reuse=True)(upper_case)
 
 
+class CustomizeRoute(BaseModel):
```

Comment:
@stevenmburns , the schema below looks good. Later, we can add validation: i) layer exists, ii) min layer is below max layer, etc.